aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -268,8 +268,41 @@ mod loader {
         }
 
         /// Override the [`HttpConnector`] used to build [`SdkConfig`](aws_types::SdkConfig).
-        pub fn http_connector(mut self, http_connector: HttpConnector) -> Self {
-            self.http_connector = Some(http_connector);
+        ///
+        /// ## Examples
+        /// ```
+        /// # #[cfg(test)]
+        /// # mod tests {
+        /// # #[test]
+        /// # fn example() {
+        /// use std::time::Duration;
+        /// use aws_smithy_client::{Client, hyper_ext};
+        /// use aws_smithy_client::erase::DynConnector;
+        /// use aws_smithy_client::http_connector::ConnectorSettings;
+        /// use aws_types::SdkConfig;
+        ///
+        /// let https_connector = hyper_rustls::HttpsConnectorBuilder::new()
+        ///     .with_webpki_roots()
+        ///     .https_only()
+        ///     .enable_http1()
+        ///     .enable_http2()
+        ///     .build();
+        /// let smithy_connector = hyper_ext::Adapter::builder()
+        ///     // Optionally set things like timeouts as well
+        ///     .connector_settings(
+        ///         ConnectorSettings::builder()
+        ///             .connect_timeout(Duration::from_secs(5))
+        ///             .build()
+        ///     )
+        ///     .build(https_connector);
+        /// let sdk_config = SdkConfig::builder()
+        ///     .http_connector(smithy_connector)
+        ///     .build();
```

Comment:
I think `SdkConfig::builder` is the wrong example to be setting, but it's not easy to include `aws-config` in the examples... Maybe we can generate a Rust code snippet as a separate file that recreates a minimal `ConfigLoader` just for examples, and include that (hidden) in each doc example that needs to illustrate it? Or just set these to `no_compile`, but that kind of stinks.

Reply:
In this case, we can alias the usage of `aws-config` so that we don't have to call it `crate`. I'll update it and see what you think.